pkgs/servers/pulseaudio/qpaeq.nix

```diff
@@ -0,0 +1,66 @@
+{ mkDerivation
+, python3
+, fetchurl
+, lib
+, pulseaudio
+}:
+
+let
+  qp = "./src/utils/qpaeq";
+in
+mkDerivation rec {
+  pname = "pulseuadio-qpaeq-${version}";
+  version = "13.0";
+
+  /* src = fetchurl {
+    url = "http://freedesktop.org/software/pulseaudio/releases/pulseaudio-${version}.tar.xz";
+    sha256 = "0mw0ybrqj7hvf8lqs5gjzip464hfnixw453lr0mqzlng3b5266wn";
+  }; */
+
+  src = ./.;
```

Comment:
This sounds wrong to me... :confused: ... reasoning?

Reply:
See above. Once that is figured out. Also it tries to build pulse from makefile, which I'm trying to avoid since I only need the qpaeq bin